flake8_spellcheck/pandas.txt

```diff
@@ -0,0 +1,61 @@
+agg
+applymap
+asfreq
+asof
+astype
+backfill
+bfill
+boxplot
+corrwith
+cov
+csv
+cummax
+cummin
+cumprod
+cumsum
+droplevel
+dropna
+dtypes
+ewm
+ffill
+fillna
+floordiv
+gbq
+groupby
+hdf
+iat
+idxmax
+idxmin
+iloc
+isin
+isna
+iterrows
+itertuples
+kurtosis
+mul
+ndim
+nlargest
+notna
+notnull
+nsmallest
+numpy
+nunique
+quantile
+radd
+rdiv
+reindex
+resample
+rfloordiv
+rmod
+rmul
+rpow
+rsub
+rtruediv
+sem
+stata
+swapaxes
+swaplevel
+truediv
+tshift
+unstack
+xarray
```

Comment:
It was suggested to me that these should also be added:

```
argmax
argmin
argsort
autocorr
backfill
barh
bfill
corr
endswith
extractall
ffill
fillna
findall
from_coo
fullmatch
hasnans
hist
idxmax
idxmin
isalnum
isalpha
isdecimal
isdigit
isin
islower
isna
isnull
isnumeric
isspace
istitle
isupper
ix
kde
kurt
ljust
lstrip
notnull
npoints
pydatetime
pytimedelta
qyear
ravel
reindex_like
rfind
searchsorted
skew
sp_values
startswith
std
strftime
swapcase
swaplevel
to_coo
to_csv
to_excel
to_hdf
to_markdown
to_sql
to_xarray
tz
unstack
var
zfill
```

what are your thoughts about adding these in?

Reply:
The words with the underscore don't need to be included if the individual parts are in the file (e.g. for to_csv we have that "csv" is in the file and "to" is in the english dictionary). Besides that it looks good.
I actually worked on a way to automate the creation of a dictionary for abitrary packages and it gets almost everything, I just have to add another layer of recursion.
The only downside is that the package needs to be installed.

When it is working, should it be implemented such that the user can specify for which packages they want a dictionary  to be created?

Reply:
> When it is working, should it be implemented such that the user can specify for which packages they want a dictionary to be created?

I'll probably need some thinking about where such a file could belong. This is tooling so it needs a bit of thought about how it gets installed.

Feel free to open a draft PR if you want to open a discussion though